src/transaction/log_writer.h

```diff
@@ -199,10 +199,6 @@ extern bool logwr_force_shutdown (void);
 extern int logwr_copy_log_file (const char *db_name, const char *log_path, int mode, INT64 start_page_id);
 extern LOG_PHY_PAGEID logwr_to_physical_pageid (LOG_PAGEID logical_pageid);
 extern const char *logwr_log_ha_filestat_to_string (enum LOG_HA_FILESTAT val);
-#if 0
```

Comment:
Please refer to the definition of these functions and change it 
```if 0```
to
``` #ifdef UNSTABLE_TDE_FOR_REPLICATION_LOG```.

Reply:
@Yechan0815 Please follow what I said. The declaration has to be in the same macro as its definitions.

Reply:
![rf](https://user-images.githubusercontent.com/12230655/185846679-25dfd710-3433-4baa-8410-25045f3add7f.png)

@wrlawodms 
I'm not sure why, but the changes aren't showing up properly here...

Reply:
Oh, now I can see. Thank you.